source/partials/next_release/_release.md.erb

```diff
@@ -13,25 +12,22 @@ This improves performance for large GoCD instances with thousands of pipelines.
 a javascript based frontend that is several thousand times faster on large instances, reducing the CPU usage of your browser. Your laptop battery will thank you. :)
 * reduces the polling rate when the dashboard is shown in a browser tab that is hidden.
 
-> Please report any issues that you observe on [GitHub issues](https://github.com/gocd/gocd/issues).
 
-
-<h4>Plugin Infrastructure Enhancements</h4>
-We have been working on a few improvements to our plugin infrastructure.
+<h4>Plugin Enhancements</h4>
```

Comment:
Plugin Enhancements implies that we are enhancing a particular plugin. Can we say "Plugin Architecture Enhancements" ?

Reply:
Or Enhancements to GoCD's Plugin Architecture